pkg/aws/client_test.go

```diff
@@ -1,13 +1,16 @@
 package aws_test
 
 import (
+	"fmt"
+	aws2 "github.com/aws/aws-sdk-go/aws"
```

Comment:
NP: same as above, just add some whitespace between the `fmt` import and the `aws2` import